pennylane/devices/qubit/apply_operation.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Functions to apply an operation to a state vector."""
-# pylint: disable=unused-argument
+# pylint: disable=unused-argument, too-many-arguments
```

Comment:
Do we need this at a module level or could we disable just for the function?

Reply:
That may be a merge artifact.